test/unit/carousel.test.js

```diff
@@ -23,95 +18,267 @@ window.resizeTo = (width, height) => {
+          value={-1}
+          infinite
+        >
+          <div/>
+          <div/>
+          <div className={'third'}/>
+        </Carousel>,
+      );
+
+      expect(wrapper.find('.BrainhubCarouselItem--active').children().hasClass('third')).toEqual(true);
+    });
+    it('Displays the last slide if the value is bigger than the number of slides', () => {
+      const wrapper = mount(
+        <Carousel
+          value={10}
+        >
+          <div/>
+          <div/>
+          <div className={'third'}/>
+        </Carousel>,
+      );
+
+      expect(wrapper.find('.BrainhubCarouselItem--active').children().hasClass('third')).toEqual(true);
+    });
+
+    it('Displays the first slide if the value is lower than 0', () => {
+      const wrapper = mount(
+        <Carousel
+          value={-10}
+        >
+          <div className={'first'}/>
+          <div/>
+          <div/>
+        </Carousel>,
+      );
+
+      expect(wrapper.find('.BrainhubCarouselItem--active').children().hasClass('first')).toEqual(true);
+    });
+  });
+  describe('dots', () => {
+    it('renders dots', () => {
+      const wrapper = setup({
+        dots: true,
+      });
+
+      expect(wrapper.find('.BrainhubCarousel__dot')).toHaveLength(3);
+    });
+
+    it('changes slide on dot click', () => {
+      const wrapper = setup({
+        dots: true,
+      });
+
+      expect(wrapper.find('.BrainhubCarouselItem').first().hasClass('BrainhubCarouselItem--active')).toEqual(true);
+
+      wrapper.find('.BrainhubCarousel__dot').at(1).simulate('click');
 
-  it('gets nearest slide index in centered carousel', () => {
-    const carousel = shallow(
-      <Carousel value={1} centered>
-        <div/>
-        <div/>
-        <div/>
-        <div/>
-      </Carousel>,
-    );
-
-    carousel.instance().setState({
-      carouselWidth: 1000,
-      dragOffset: -2500,
-    });
-    expect(carousel.instance().getNearestSlideIndex()).to.equal(3);
+      expect(wrapper.find('.BrainhubCarouselItem').at(1).hasClass('BrainhubCarouselItem--active')).toEqual(true);
+    });
   });
+  describe('arrows', () => {
+    it('renders arrows', () => {
+      const wrapper = setup({
+        arrows: true,
+      });
+
+      expect(wrapper.find('.BrainhubCarousel__arrows')).toHaveLength(2);
+    });
 
-  it('gets current slide index', () => {
-    const carousel = shallow(
-      <Carousel value={2}>
-        <div/>
-        <div/>
-        <div/>
-        <div/>
-      </Carousel>,
-    );
-    expect(carousel.instance().getCurrentValue()).to.equal(2);
+    it('changes slide on active arrow click', () => {
+      const wrapper = setup({
+        arrows: true,
+      });
+
+      expect(wrapper.find('.BrainhubCarouselItem').first().hasClass('BrainhubCarouselItem--active')).toEqual(true);
+
+      wrapper.find('.BrainhubCarousel__arrowRight').simulate('click');
+
+      expect(wrapper.find('.BrainhubCarouselItem').at(1).hasClass('BrainhubCarouselItem--active')).toEqual(true);
```

Comment:
is `.toEqual(true)` needed?
it doesn't sound like a valid human phrase

Reply:
Change to toBeTruthy